addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/internal/ColorUtil.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mihome.internal;
+
+/**
+ * Color utilities.
+ *
+ * @author paulkaplan
```

Comment:
reading the comment
```
        // Used this: https://gist.github.com/paulkaplan/5184275 at the beginning
        // based on http://stackoverflow.com/questions/7229895/display-temperature-as-a-color-with-c
        // this answer: http://stackoverflow.com/a/24856307
        // (so, just interpretation of pseudocode in Java)
```
it seems @pboos wrote the code but wanted to give credits to the guy who showed off a similar implementation in JS. 
@pboos please clarify!
Can we adjust this to you as author?